examples/datastore/datastore-BigQuery.tf

```diff
@@ -0,0 +1,58 @@
+provider "satori" {
+  #can be provided via environment variable: SATORI_SA
+  service_account     = "aaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaaa"
+  #can be provided via environment variable: SATORI_SA_KEY
+  service_account_key = "bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb"
+  #satori account id for all resources in this terraform
+  satori_account      = "cccccc-cccc-cccc-cccc-cccccccccccc"
+  url        = "https://<satori-mgmt-server-address>:8014"
+  verify_tls = false
+}
+locals {
+  dataaccess_controller_id = "<assigned dataaccess_controller_id>"
+}
+
+resource "satori_datastore" "datastore0" {
+  name = "exampleDatastore"
+
+  dataaccess_controller_id = local.dataaccess_controller_id
+  # data source specific connection settings
+  type                     = "BIGQUERY"
+  project_ids              = [111, 222] #  BigQuery affected project ids
```

Comment:
Is't should be strings and not integers?

Reply:
correct - it's still comverts to string

Reply:
changed the example

Reply:
I don't see the change :), project ids are not ids, they are actually names :)